gdpr/anonymizers/model_anonymizers.py

```diff
@@ -292,36 +299,80 @@ def anonymize_qs(self, qs: QuerySet) -> None:
     def deanonymize_qs(self, qs: QuerySet) -> None:
         raise NotImplementedError()
 
+    def get_anonymizer_or_none(self, name):
+        return self.anonymizers.get(name, None)
+
+    def update_related_anonymizer_fields(self, field_name:str, obj: Model, related_fields:Fields,
+                                         legal_reason: Optional[LegalReason] = None,
+                                         purpose: Optional["AbstractPurpose"] = None,
+                                         anonymization: bool = True):
```

Comment:
the type hinting makes such a mess, how about putting each argument on a separate line?

Reply:
I would appreciate docstring for this non trivial method.

Reply:
@Formulka I am afraid it's convention of this lib :( there are many methods written like this.